test/test_rosdep_source.py

```diff
@@ -56,6 +56,47 @@ def get_test_dir():
     return os.path.abspath(os.path.join(os.path.dirname(__file__), 'source'))
 
 
+def get_simple_script_exit_with(exitcode):
```

Comment:
This whole block of functions is duplicated across two test files and should be deduplicated.